### DISCUSSION TITLE
dbh_pc calculation for already normalized point cloud

### DISCUSSION THREAD

**Discussion by SyedTuhinAli** [lmterryn/ITSMe#7] [General]
Thanks for this nice package. Would a quick fix be to put lowest_point = 0 if a single tree point cloud is already normalized if I understand the function correctly? I have normalized the tree point cloud, where some tree bases are missing in the tree trunk. Therefore, calculating from lowest_point to 1.3m would be wrong for me. Please let me know if I am wrong here

**Comment by lmterryn** [comment]
Yes, you can give the lowest point you choose as an input for the "dtm" input parameter.
This needs to be a dataframe with x,y,z (like what you get when you read a tree point cloud).


________________________________
Van: sta107 ***@***.***>
Verzonden: woensdag 6 november 2024 17:33
Aan: lmterryn/ITSMe ***@***.***>
CC: Subscribed ***@***.***>
Onderwerp: [lmterryn/ITSMe] dbh_pc calculation for already normalized point cloud (Discussion #7)


Would a quick fix be to put lowest_point = 0 if a single tree point cloud is already normalized if I am understanding the function correctly? I have normalized tree point cloud where some tree bases are missing in the tree trunk. Therefore, calculating from lowest_point to 1.3m would be wrong for me. please let me know if I am wrong here
